data/CRMEntity.php

```diff
@@ -3758,7 +3683,7 @@ public function getOrderBy() {
 		$order_by = '';
 		$customView = new CustomView($cmodule);
 		$viewid = $customView->getViewId($cmodule);
-		$sortfieldbyfirst = cbCVManagement::getFieldValuesByCvId($viewid)['sortfieldbyfirst'];
```

Comment:
I fixed this warning a few days ago like this:  https://github.com/tsolucio/corebos/commit/2e0a1cbf43b71fed827c8bf90e517d25883a3552   which is correct/better?